src/test/java/com/powsybl/openloadflow/sensi/ac/AcSensitivityAnalysisContingenciesTest.java

```diff
@@ -428,7 +429,57 @@ void testBusVoltagePerTargetVFunctionRef() {
         assertEquals(0.987d, getFunctionReference(contingencyValue, busIds.get(7)), LoadFlowAssert.DELTA_V);
         assertEquals(0.989d, getFunctionReference(contingencyValue, busIds.get(8)), LoadFlowAssert.DELTA_V);
         assertEquals(1d, getFunctionReference(contingencyValue, busIds.get(9)), LoadFlowAssert.DELTA_V);
+    }
 
+    @Test
+    void testHvdcSensiRescale() {
+        double sensiChange = 10e-4;
+        // test injection increase on loads
+        Network network = HvdcNetworkFactory.createNetworkWithGenerators();
+        network.getGeneratorStream().forEach(gen -> gen.setMaxP(2 * gen.getMaxP()));
+        SensitivityAnalysisParameters sensiParameters = createParameters(false, "b1_vl_0", true);
+        sensiParameters.getLoadFlowParameters().getExtension(OpenLoadFlowParameters.class).setSlackBusSelector(new MostMeshedSlackBusSelector());
+        List<Pair<String, String>> variableAndFunction = List.of(
+            Pair.of("hvdc34", "l12"),
+            Pair.of("hvdc34", "l13"),
+            Pair.of("hvdc34", "l23"),
+            Pair.of("hvdc34", "l25"),
+            Pair.of("hvdc34", "l45"),
+            Pair.of("hvdc34", "l46"),
+            Pair.of("hvdc34", "l56")
+        );
+
+        Network network1 = HvdcNetworkFactory.createNetworkWithGenerators();
+        network1.getGeneratorStream().forEach(gen -> gen.setMaxP(2 * gen.getMaxP()));
+        network1.getLine("l25").getTerminal1().disconnect();
+        network1.getLine("l25").getTerminal2().disconnect();
+        runLf(network1, sensiParameters.getLoadFlowParameters());
+        Network network2 = HvdcNetworkFactory.createNetworkWithGenerators();
+        network2.getHvdcLine("hvdc34").setActivePowerSetpoint(network1.getHvdcLine("hvdc34").getActivePowerSetpoint() + sensiChange);
+        network2.getGeneratorStream().forEach(gen -> gen.setMaxP(2 * gen.getMaxP()));
+        network2.getLine("l25").getTerminal1().disconnect();
+        network2.getLine("l25").getTerminal2().disconnect();
+        runLf(network2, sensiParameters.getLoadFlowParameters());
+        Map<String, Double> loadFlowDiff = network.getLineStream().map(line -> line.getId())
+            .collect(Collectors.toMap(
+                lineId -> lineId,
+                line -> (network1.getLine(line).getTerminal1().getP() - network2.getLine(line).getTerminal1().getP()) / sensiChange
+            ));
+
+        HvdcWriter hvdcWriter = HvdcWriter.create();
+        SensitivityFactorReader reader = createHvdcReader(variableAndFunction);
+        sensiParameters.getLoadFlowParameters().getExtension(OpenLoadFlowParameters.class).setSlackBusSelector(new NameSlackBusSelector("b1_vl_0")); // the most meshed bus selected in the loadflow
+        sensiProvider.run(network, VariantManagerConstants.INITIAL_VARIANT_ID, Collections.singletonList(new Contingency("l25", new BranchContingency("l25"))),
+            sensiParameters, reader, hvdcWriter);
+
+        // FIXME
```

Comment:
I have remplaced LCCs by VSCs but it still does not work with distributed slack.